### ISSUE TITLE
deploy errors

### ISSUE THREAD

**Issue by micahmicah** [risd/risd-graduate-welcome#1]
When attempting to run `wh deploy` I encountered a token error.

![screen shot 2017-01-09 at 1 59 33 pm](https://cloud.githubusercontent.com/assets/1166088/21779031/dce88136-d673-11e6-81dc-fb5515db810e.png)

**Comment by rubillionaire** [comment]
Ah I've run into one of these. Token error means it is not under hosted webhook.org. It looks like this is on `risd.systems`

I will take a closer look at the issue tomorrow, until then I might try the `rm-wh` version of the commands. including the `init`, which will pull from your computers local config (`~/.risdmedia/wh.json`)

**Comment by micahmicah** [comment]
Ah yes, I knew I was doing something wrong. It looks like I got a successful deploy 
 
![screen shot 2017-01-10 at 10 24 46 am](https://cloud.githubusercontent.com/assets/1166088/21811835/09f2c8ac-d71f-11e6-84f4-02460c3bbb06.png)

But the templates on the live version of the site don't seem to reflect what I have locally.

On systems:
![screen shot 2017-01-10 at 10 25 34 am](https://cloud.githubusercontent.com/assets/1166088/21811856/25de958c-d71f-11e6-9133-b2be32ff1462.png)

On local:
![screen shot 2017-01-10 at 10 25 36 am](https://cloud.githubusercontent.com/assets/1166088/21811876/2b676b6e-d71f-11e6-964c-a7afcfeb3b9d.png)

My questions are, is this a branch issue? I'm currently on a feature branch to explore a particular content type structure. Or is it a build time issue? I just uploaded the templates and am wondering if this will take an additional 15–20 minutes to update the templates on `systems`.
